package.json

```diff
@@ -180,4 +180,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
Probably an "ensure newline at tend of file" rule from my IDE that got triggered at some point.